src/Card/Card.js

```diff
@@ -1,17 +1,15 @@
 import './Card.css';
+import { NavLink } from 'react-router-dom'
 
 const Card = ({ park }) => {
-
   return (
-    <div className='card'>
-      <div className='img-box'>
-        <img src={park["images"][0]["url"]} className='img' alt='park image'/>
-      </div>
+    <NavLink to={`/${park.parkCode}`} className='card'>
+      <img src={park.images?.[0]?.url} className='img' alt='park image' />
```

Comment:
This will not handle urls listed in the data that have been removed! That's why you're seeing the 404s in the console.  I think you can handle this with the `onError` listener along the lines of:

```
<img onError={imageOnError} src={park.images?.[0]?.url} className='img' alt='park image' /> 

const imageOnError = (event) => {
    event.currentTarget.src = <path to your broken image>;
};
```